packages/remix-dev/config.ts

```diff
@@ -493,7 +493,7 @@ export async function resolveConfig(
       pkgJson.update({
         dependencies: {
           ...pkgJson.content.dependencies,
-          isbot: "latest",
+          isbot: "^4",
```

Comment:
going forward, `remix dev` will add v4 to the `package.json` instead of `latest` - this is the primary "fix"